freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/dns/model/DnsRecordRegexpPatterns.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.freeipa.api.v1.dns.model;
+
+public final class DnsRecordRegexpPatterns {
+
+    public static final String DNS_ZONE_PATTERN = "^[a-zA-Z0-9-\\.]+$";
```

Comment:
I think this might be the same rules as above: "^[a-zA-Z0-9]+[a-zA-Z0-9.-]*[a-zA-Z0-9.]+$"